server/backend/test/steps/send-notification.steps.ts

```diff
@@ -151,27 +179,39 @@ defineFeature(feature, (test) => {
     let sendRes: request.Response;
 
     given("I have camera A's credentials", async () => {
-      const { camera, keyPair } = createCameraWithKeyPair('Camera-A', 'Serial-A');
+      const { camera, keyPair } = createCameraWithKeyPair(
+        'Camera-A',
+        'Serial-A',
+      );
       cameraA = camera;
       cameraA.group = new Group('group');
       cameraA.group.user = new User();
+      cameraA.group.user.expoToken = 'ExponentPushToken[000000000000]';
       await cameraRepository.persistAndFlush(cameraA);
       token = getCameraAuthToken(cameraA, keyPair.privateKey);
     });
-    and('Camera A has 1 notification', async () => {
-      cameraA.notifications.add(new Notification());
-      await cameraRepository.flush();
+    and('Camera A has 0 notifications', async () => {
+      const resBefore = await request(app.getHttpServer())
+        .get(`/cameras/${cameraA.id}/notifications`)
+        .set('Authorization', token);
+      expect(resBefore.status).toBe(200);
+      expect(resBefore.body).toHaveLength(0);
     });
     when('I try to send a notification on behalf of camera A', async () => {
       sendRes = await request(app.getHttpServer())
         .put(`/cameras/${cameraA.id}/notifications`)
-        .set('Authorization', token);
+        .set('Authorization', token)
+        .send();
```

Comment:
Why a random send here?

Reply:
I thought it was needed for the request to actually go through, but it seems to still pass locally after removing it. I'll remove and push